wondrous-app/components/Settings/styles.tsx

```diff
@@ -227,6 +227,32 @@ export const SettingsHeaderText = styled(Typography)`
   }
 `;
 
+export const SettingsHeaderInviteButton = styled.button`
+  display: flex;
+  align-items: center;
+  gap: 12px;
+  background: #232323;
+  color: #ffffff;
+  font-size: 15px;
+  font-weight: 700;
+  line-height: 150%;
+  padding: 7px 7px 7px 14px;
+  outline: none;
+  border: none;
+  border-radius: 1000px;
+  cursor: pointer;
+`;
+
+export const SettingsHeaderInviteButtonIcon = styled.div`
```

Comment:
Can you use this common button `components/Button/index.tsx` or create new one to prevent duplicated code?